src/test/java/org/fcrepo/importexport/exporter/ExporterTest.java

```diff
@@ -370,53 +373,6 @@ public void testExportBagCustomTags() throws Exception, FcrepoOperationFailedExc
         Assert.assertTrue(customLines.contains("Foo : Bar"));
         Assert.assertTrue(customLines.contains("Baz : Quux"));
     }
-
-    @Test
-    public void testExportVersionsContainers() throws Exception {
```

Comment:
The test you referenced was copy and pasted from this method, then cleaned up to use the helper methods provided in ExporterVersionTest and match the style of the other test methods.  I had written ExporterVersionTest to contain all the versioning tests.  Both define the same resources and same assertions, I just forgot to remove this one from its original home when doing refactoring.